Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,49 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_tabletop_rping",
+            category=['monika','literature'],
+            prompt="Tabletop Roleplaying",
+            random=True,
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+label monika_tabletop_rping:
+    m "Say, [player], there's this literature genre that I've been meaning to bring up lately."
+    m "To be honest, I've been intrigued by it for quite some time, but never got the chance to participate."
+    m "Just like Doki Doki Literature Club, it's kind of a mix between a game and a form of literature."
+    m "It's called tabletop roleplaying! Have you heard of it?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "It's called tabletop roleplaying! Have you heard of it?"
+        "Yes.":
+            m "That's great!"
+
+        "No.":
+            m "Oh really? It's really quite interesting you know."
+
+    m "You may have played 'pretend' when you were a kid growing up, right?"
```

Comment:
Isn't `'when you were a kid growing up'` a tautology?
I'd say either `'when you were a kid'` or `'when you were growing up'` (I like the first one better).

Reply:
Actually, I disagree that it's a tautology, but I see where you're coming from. The first part, being a kid, is different in that it established the age range, the second part "growing up", establishes the direction your age changes. Being a kid is a static phrase, while growing up is a moving term.

Kind of like saying "When you were a car driving". Growing up is not a necessity of being a kid without taking time into consideration.